arex-storage-web-api/src/main/java/com/arextest/storage/repository/RepositoryProviderFactory.java

```diff
@@ -53,6 +57,20 @@ public MockCategoryType findCategory(String categoryName) {
     return null;
   }
 
+  public Set<MockCategoryType> getCategoryTypesByName(String[] categoryList) {
+    if (ArrayUtils.isEmpty(categoryList)) {
+      return categoryTypes;
+    }
+
+    return Arrays.stream(categoryList)
+        .map(categoryName -> categoryTypes.stream()
+            .filter(categoryType -> categoryName.equals(categoryType.getName()))
+            .findFirst()
+            .orElse(null))
+        .filter(Objects::nonNull)
```

Comment:
use `findCategory' directly without repeating the code

Reply:
ok，and i optimized the use of findCategory and constructed a map object to store mockcategorytype objects to avoid looping to find matching values.